e2e/e2e_test.go

```diff
@@ -57,20 +57,8 @@ var _ = Describe("Mock test", func() {
 
 	Context("E2E with Error Cases", func() {
 		When("Config error", func() {
-			It("should hit usage", func() {
```

Comment:
so... is there any usage hint if/when user use tape cli in wrong way?

Reply:
As the It should show usage as hit?

Reply:
If the user use tape cli with undefined subcommand, the program will exit and output some help information.
  ```
./tape runs
  tape: error: unexpected runs, try --help
```
If the user does not specify any sub-commands or parameters, the program will execute the default configuration.
So I just keep the unit test that the config file does not exist and remove others. If other test cases are still meaningful, I will add them again.

Reply:
the original test case here, is to show some thing when user give error parameters.
in this case, I would like to suggest we adding two cases below:
1. `./tape invalided ` and expect `tape: error: unexpected runs, try --help`
2. `./tape --help or tape -h`

Reply:
for better UX from CLI.

Reply:
I added four tests for "command error" type. Please review them.